docker-build/9.0.0.10/Dockerfile.offline

```diff
@@ -0,0 +1,58 @@
+# build image to extract the tar ball.
+FROM ubuntu:16.04 as builder
+ ARG WAS_VERSION=9.0.0.10
+ ARG WAS_TAR_BALL=was-${WAS_VERSION}.tar.gz
+COPY im/${WAS_TAR_BALL} /
+RUN tar -xzf "$WAS_TAR_BALL"
+# patch wsadmin.sh to avoid error when deploying apps
+RUN sed -i 's/-Xms256m/-Xms1024m/g' /opt/IBM/WebSphere/AppServer/bin/wsadmin.sh \
+    && sed -i 's/-Xmx256m/-Xmx1024m/g' /opt/IBM/WebSphere/AppServer/bin/wsadmin.sh
+
+
+# final image that will copy extracted tar ball from builder
+FROM ubuntu:16.04
+ARG USER=was
+ARG GROUP=was
+
+
+ARG PROFILE_NAME=AppSrv01
+ARG CELL_NAME=DefaultCell01
+ARG NODE_NAME=DefaultNode01
+ARG HOST_NAME=localhost
+ARG SERVER_NAME=server1
+ARG ADMIN_USER_NAME=wsadmin
+
+COPY scripts/ /work/
+COPY licenses /licenses/
+
+
+RUN apt-get update && apt-get install -y openssl
+
+COPY scripts/ /work/
```

Comment:
is this a dup of line 25?

Reply:
Yes, it was. Just corrected that on a new commit.